tools/mpremote/mpremote/commands.py

```diff
@@ -361,8 +405,8 @@ def do_filesystem(state, args):
         # leading ':' if the user included them.
         paths = [path[1:] if path.startswith(":") else path for path in paths]
 
-    # ls implicitly lists the cwd.
-    if command == "ls" and not paths:
+    # ls and tree implicitly lists the cwd.
+    if command in ["ls", "tree"] and not paths:
```

Comment:
Please use tuple `("ls", "tree")`.

Reply:
I'd swear I had done that before.
Sorry to make you repeat yourself.

Reply:
You did already do that, on a separate line :smile: